stdlib/2/os/path.pyi

```diff
@@ -4,7 +4,8 @@
 # based on http://docs.python.org/3.2/library/os.path.html
 # adapted for 2.7 by Michal Pokorny
 
-from typing import overload, List, Any, Tuple, BinaryIO, TextIO, TypeVar, Callable, AnyStr
+from typing import overload, List, Any, Tuple, BinaryIO, TextIO, TypeVar, Callable, AnyStr, Union
+EitherStr = Union[bytes, unicode]
```

Comment:
You can "import it" from other stubs. Those imports specifically allow the `from X import *` form and we use underscored names to make it obvious those are not part of the API. Existing usage in typeshed overwhelmingly uses this pattern: https://gist.github.com/ambv/ef061793d5c10169c455e8630a43e8df